modules/caddyhttp/fileserver/browse.go

```diff
@@ -68,6 +68,10 @@ type Browse struct {
 	SortOptions []string `json:"sort,omitempty"`
 }
 
+var (
+	maxDirLimit = 10000
+)
```

Comment:
This should be a constant, not a variable. Should have a prefix like `default`

Reply:
Done.